src/plugins/controls/server/options_list/options_list_cluster_settings_route.ts

```diff
@@ -39,6 +40,17 @@ export const setupOptionsListClusterSettingsRoute = ({ http }: CoreSetup) => {
           },
         });
       } catch (e) {
+        if (e instanceof errors.ResponseError && e.body.error.type === 'security_exception') {
+          /**
+           * in cases where the user does not have the 'monitor' permission this check will fail. In these cases, we will
+           * fall back to assume that the allowExpensiveQueries setting is on, because it defaults to true.
+           */
+          return response.ok({
+            body: {
+              allowExpensiveQueries: 'true',
+            },
```

Comment:
Woops! Left my nit comment behind somehow 😆 Here it is...

**nit:** Should this be: 

```suggestion
            body: {
              allowExpensiveQueries: true,
            },
```

? Doesn't make a difference in the behaviour, since it is parsed as a `boolean` value later in the options list service, but just for consistency with the non-error state (`allowExpensiveQueries: allowExpensiveQueries === 'true'`)

Reply:
Oh very true, I got tripped up by the if statement being `=== 'true'` - which of course would return a boolean. Will update this!